pkgs/development/tools/dobi/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "dobi-${version}";
+  version = "v0.8";
```

Comment:
Version strings must begin with a digit. In this case, the package will end up with the name "dobi-v0.8" and no version (see `builtins.parseDrvName`).